.travis.yml

```diff
@@ -91,24 +91,6 @@ jobs:
     - <<: *rust
       rust: nightly
       if: (type != cron) AND (branch != staging.tmp) AND (branch != trying.tmp)
-    # FIXME: needs recompiling with mock flags, as integration tests use tx-query that does remote attestation
-    - name: Integration Test
-      language: node_js
-      node_js: 10
-      sudo: required
-      dist: trusty
-      if: (type != cron) AND ((branch = staging) OR (branch = trying))
-      script:
-        - cd integration-tests
-        - ./prepare.sh || travis_terminate 1;
-        - . ./env.sh
-        - docker-compose up -d || travis_terminate 1;
-        - ./wait-for-setup.sh || (docker-compose ps; docker-compose logs -t --tail="all"; travis_terminate 1);
-        - ./run-test.sh || (docker-compose ps; docker-compose logs -t --tail="all"; travis_terminate 1);
-        - |
-          cd client-rpc
-          yarn
-          yarn test || (docker-compose ps; docker-compose logs -t --tail="all"; travis_terminate 1);
```

Comment:
I suggest to keep running integration tests in mock/SW mode on TravisCI to ensure mock/SW mode is working fine.

Reply:
We can support the mock mode in another PR, maybe another pipeline in drone?
The advantage of travis is able to run in free cloud?

Reply:
Yes, we can do it in another PR. It is not necessarily TravisCI, can be another pipeline in drone as well.

Reply:
> Yes, we can do it in another PR. It is not necessarily TravisCI, can be another pipeline in drone as well.

I think since it doesn't need the sgx stuff in mock mode, it might quite easy to set up in Travis too, need to get familiar with the syntax though.